lib/cea/mp4_cea_parser.js

```diff
@@ -45,6 +47,11 @@ shaka.cea.Mp4CeaParser = class {
      * @private {number}
      */
     this.defaultSampleSize_ = 0;
+
+    /*
+     * @private {boolean}
+     */
+    this.isHEVC_ = baseCodec == 'hevc' || baseCodec == 'dovi';
```

Comment:
Since there seem to be multiple codec strings for this, could we deduce HEVC instead from the presence of a certain box in the init segment?

For example, picking apart an h264 init segment with https://thumb.co.il/, I see trak > mdia > minf > stbl > stsd > sampleDescriptions > avc1.  At the same place in the hierarchy, for h265 content I see a hev1 box instead of an avc1 box.